chat-plugins/info.js

```diff
@@ -2335,3 +2335,14 @@ var commands = exports.commands = {
 	},
 	htmlboxhelp: ["/htmlbox [message] - Displays a message, parsing HTML code contained. Requires: ~ # with global authority"]
 };
+
+process.nextTick(function () {
```

Comment:
This is the only file that needs this data, and it's a plugin; so I would say "yes".